otbenchmark/ReliabilityProblem60.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue May  5 12:56:49 2020
+
+@author: Jebroun
+
+Class to define the ReliabilityProblem60 benchmark problem.
+"""
+
+from otbenchmark.ReliabilityBenchmarkProblem import ReliabilityBenchmarkProblem
+import openturns as ot
+
+
+class ReliabilityProblem60(ReliabilityBenchmarkProblem):
+    def __init__(
+        self,
+        threshold=0.0,
+        mu1=2200,
+        sigma1=220,
+        mu2=2100,
+        sigma2=210,
+        mu3=2300,
+        sigma3=230,
+        mu4=2000,
+        sigma4=200,
+        mu5=1200,
+        sigma5=480,
+    ):
+        """
+        Creates a reliability problem RP60.
+
+        The event is {g(X) < threshold} with
+
+        you can use this line :
+
+            "https://rprepo.readthedocs.io/en/latest/reliability_problems.html#rp60"
+
+        We have :
+            x1 ~ LogNormalMuSigma(mu1, sigma1)
+
+            x2 ~ LogNormalMuSigma(mu2, sigma2)
+
+            x3 ~ LogNormalMuSigma(mu3, sigma3)
+
+            x4 ~ LogNormalMuSigma(mu4, sigma4)
+
+            x5 ~ LogNormalMuSigma(mu5, sigma5)
+
+        Parameters
+        ----------
+        threshold : float
+            The threshold.
+        mu1 : float
+            The mean of the underlying gaussian distribution of X1.
```

Comment:
I think this is wrong. The description is: "The mean of the LogNormal random variable X1."